ghttp/ghttpProcess.c

```diff
@@ -1247,7 +1247,7 @@ static int ghiParseChunkSize
 {
 	char * header;
 	int len;
-	int num;
+	unsigned num;
```

Comment:
use "unsigned int" not unsigned

Reply:
https://github.com/GameProgressive/UniSpySDK/pull/27/commits/bd64fbac79ff64d0bf18e90899d269f6d4401f50